components/automate-cli/cmd/chef-automate/automateClusterCtlUtil.go

```diff
@@ -159,3 +159,29 @@ func executeSecretsInitCommand(secretsKeyFilePath string) error {
 	}
 	return nil
 }
+
```

Comment:
Can we change the filenaming conventions used? I believe across Automate repo we have used delimit separated multi word convention with `-` or `_`

Reply:
We had converstation on file naming convention with Vivek Yadav long back, and we had finalised to use camline case for the same.